example_allocators/bitmap_allocator/good_client.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * good_client.c
+ * Jeremy Singer
+ *
+ * Example use of bitmap allocator, showing how
+ * 'good' client code should interact with the
+ * allocator.
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "bitmap_alloc.h"
+// #include "include/common.h"
+
+/* should program generate debugging output? 1 for Yes, 0 for No */
+#define DEBUG_PRINTF 1
+
+/* size of buffer to allocate (in list cells) */
+#define NUM_CELLS 64
+
+/* trivial linked list data structure */
+typedef struct _list
+{
+	int payload;
+	struct _list *next;
+	;
+} list;
+
+int main()
+{
+	unsigned int i = 0;
+	unsigned int n = 1;
+	list *l = NULL;
+
+	init_alloc(NUM_CELLS, sizeof(list));
+
+	while (n <= NUM_CELLS)
+	{
+
```

Comment:
Extraneous blank line.

Reply:
Done thanks!

Reply:
Can you push your fixed commits (not force pushing!) and say "Fixed in XYZ" where "XYZ" is the git commit? Thanks!

Reply:
Fixed in `7906aad`.